### DISCUSSION TITLE
How to download member only youtube videos?

### DISCUSSION THREAD

**Discussion by stefaneliu** [mxpv/podsync#558] [General]
Hello.
I am a youtube channel member and I would like to download the playlist.
I configed my account API in config.toml.
But I failed to download the member only videos.
In log file, it says:

[youtube] 23QyfKWUqPI: This video is available to this channel's members on level: XXXXX (or any higher level). Join this channel to get access to members-only content and other exclusive perks. 

I am frustrated in how to download it. It is ridiculous because I really am the member of the channel. Would you please help me?
Many thanks.

**Comment by tiwarimritunjay3723-hue** [comment]
https://youtu.be/kw7BMSt87Mw?si=cQzfjepBq51C5qSo

**Comment by bw4518** [comment]
That will probably require use of https://github.com/yt-dlp/yt-dlp/wiki/Extractors#exporting-youtube-cookies
